tests/test_cache.py

```diff
@@ -0,0 +1,46 @@
+import redis
+import unittest
+from cachecontrol.caches.file_cache import FileCache
+from cachecontrol.caches.redis_cache import RedisCache
+from canonicalwebteam.http import CachedSession
+from datetime import timedelta
+from unittest import TestCase
+
+
+def create_redis_connection_pool(
+    redis_port="6379", redis_host="localhost", redis_db=0
+):
+    return redis.ConnectionPool(host=redis_host, port=redis_port, db=redis_db)
```

Comment:
What if I don't run redis locally?

Can you check if you can mock redis to run the tests?